.github/workflows/remove-packages.yml

```diff
@@ -0,0 +1,71 @@
+name: remove-packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      packages:
+        description: 'Packages to remove'
+        required: true
+      debug_with_ssh_key:
+        description: 'Public SSH key to use to debug failures'
+        required: false
+
+env:
+  GIT_CONFIG_PARAMETERS: "'user.name=Git for Windows Build Agent' 'user.email=ci@git-for-windows.build' 'windows.sdk64.path=' 'windows.sdk32.path=${{ github.workspace }}' 'http.sslbackend=schannel' 'core.autocrlf=false' 'checkout.workers=16'"
+  HOME: "${{ github.workspace }}\\home\\git-ci"
+  MSYSTEM: MSYS
+
+jobs:
+  remove-packages:
+    if: github.repository_owner == 'git-for-windows' || github.event.inputs.debug_with_ssh_key != ''
+    runs-on: windows-latest
+    environment: sync
+    steps:
+      - name: clone git-sdk-32
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: true
+          token: ${{ secrets.PUSH_TOKEN }}
+      - name: use git-sdk-32's Bash and git.exe
+        run: "usr\\bin\\bash.exe -lc 'cygpath -aw /usr/bin >>$GITHUB_PATH && cygpath -aw /cmd >>$GITHUB_PATH'"
+      - name: remove packages
+        shell: bash
+        env:
+          pkgs: ${{ inputs.packages }}
```

Comment:
In contrast to Azure Pipelines, there is no way to inspect the input parameters for failed workflow runs. For example, you cannot see what `packages` you provided in https://github.com/jeremyd2019/git-sdk-32/actions/runs/13379109030/job/37364348987.

To remedy this, in Git for Windows' workflows I like to mirror all workflow dispatch inputs in a global environment block. That way, at least expanding the first step's first lines will show what the parameters were (see e.g. https://github.com/git-for-windows/git-for-windows-automation/actions/runs/13378287807/job/37362013059#step:2:17, which shows what inputs that particular failed run received).

Reply:
https://github.com/jeremyd2019/git-sdk-32/actions/runs/13379299665/job/37364893203#step:2:19